src/common/components/NoticeSlider/index.tsx

```diff
@@ -0,0 +1,43 @@
+'use client';
+
+import { useRef } from 'react';
+import Slider from 'react-slick';
+
+import { Text } from '..';
+
+const setting = {
+  arrows: false,
+  dots: false,
+  autoplay: true,
+  infinite: true,
+  speed: 700,
+  slidesToShow: 1,
+  slidesToScroll: 1,
+};
+
+type NoticeSliderProp = {
+  images: string[];
+  onClick: VoidFunction;
+};
+
+export const NoticeSlider = ({ images, onClick }: NoticeSliderProp) => {
+  const slider = useRef<Slider>(null);
+  return (
+    <Slider ref={slider} className="w-full" {...setting}>
+      {images?.map((url, idx) => (
+        <div onClick={onClick} className="relative h-auto rounded-12px">
```

Comment:
```suggestion
        <a tagret="_blank" rel="noreferrer noopener"  href={url} no className="relative h-auto rounded-12px">
```

하이퍼링크만 적용될 것 같은데 button 보다는 a tag가 접근성 상 좋지 않을까 싶습니답

Reply:
오 맞네요 핸들러 매번 작성하기보단 url props로 처리하는게 편리할 것 같네용

Reply:
[11f5a07](https://github.com/depromeet/toks-web/pull/404/commits/11f5a074c531bf292603c9ec4232052bef22f687) 반영했습니다 ~!